src/Phinx/Db/Adapter/MysqlAdapter.php

```diff
@@ -1036,8 +1036,12 @@ public function getPhinxType($sqlTypeDef)
                     break;
             }
 
-            // Call this to check if parsed type is supported.
-            $this->getSqlType($type, $limit);
+            try {
+                // Call this to check if parsed type is supported.
+                $this->getSqlType($type, $limit);
+            } catch (\RuntimeException $e) {
```

Comment:
I don't like this try catch, but `getSqlType` in other adapters throw a `RuntimeException`. Changing this behaviour in the `MysqlAdapter` isn't consistent so I decided to use this way.

Reply:
Yeah, it would be nicer if the exception was more specific

Reply:
I'll add a more specific exception.